traffic_control/models.py

```diff
@@ -160,3 +176,101 @@ class Meta:
+    code = models.ForeignKey(TrafficSignCode, on_delete=models.CASCADE)
+    value = models.IntegerField(_("Traffic sign code value"), blank=True, null=True)
+    structure_id = models.IntegerField(_("Structure id"), blank=True, null=True)
+    structure_type = models.CharField(
+        max_length=10, choices=Structure.choices, default=Structure.OTHER
+    )
+    lifecycle = models.ForeignKey(Lifecycle, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(_("Created at"), auto_now_add=True)
+    updated_at = models.DateTimeField(_("Updated at"), auto_now=True)
+    deleted_at = models.DateTimeField(_("Deleted at"), blank=True, null=True)
+    created_by = models.ForeignKey(
+        get_user_model(),
+        related_name="created_by_traffic_sign_real_set",
+        on_delete=models.CASCADE,
+    )
+    updated_by = models.ForeignKey(
+        get_user_model(),
+        related_name="updated_by_traffic_sign_real_set",
+        on_delete=models.CASCADE,
+    )
+    deleted_by = models.ForeignKey(
+        get_user_model(),
+        related_name="deleted_by_traffic_sign_real_set",
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True,
+    )
+    validity_period_start = models.DateField(
+        _("Validity period start"), blank=True, null=True
+    )
+    validity_period_end = models.DateField(
+        _("Validity period end"), blank=True, null=True
+    )
+    seasonal_validity_period_start = models.DateField(
+        _("Seasonal validity period start"), blank=True, null=True
+    )
+    seasonal_validity_period_end = models.DateField(
+        _("Seasonal validity period end"), blank=True, null=True
+    )
+    owner = models.CharField(_("Owner"), max_length=254, blank=True, null=True)
+    manufacturer = models.CharField(
+        _("Manufacturer"), max_length=254, blank=True, null=True
+    )
+    rfid = models.CharField(_("RFID"), max_length=254, blank=True, null=True)
+    txt = models.CharField(_("Txt"), max_length=254, blank=True, null=True)
+    installation_date = models.DateField(_("Installation date"))
+    installation_status = models.CharField(
+        max_length=10,
+        choices=InstallationStatus.choices,
+        default=InstallationStatus.ACTIVE,
+    )
+    installation_id = models.CharField(_("Installation id"), max_length=254)
+    installation_details = models.CharField(
+        _("Installation details"), max_length=254, blank=True, null=True
+    )
+    condition = models.IntegerField(choices=Condition.choices, default=Condition.GOOD)
+    decision_id = models.CharField(_("Decision id"), max_length=254)
+    size = models.CharField(max_length=1, choices=Size.choices, default=Size.MEDIUM)
+    reflection_class = models.CharField(
+        max_length=2, choices=Reflection.choices, default=Reflection.R1
+    )
+    surface_class = models.CharField(
+        max_length=6, choices=Surface.choices, default=Surface.DIRECT
```

Comment:
Not strictly related to this PR but does `Surface.DIRECT` indicate that the sign is not curved in any way? Are these terms specified by someone else or how these are determined?

I'm fairly sure that when speaking of surfaces, "direct" is not the proper term to say that it is not curved or crooked in any way. I'd say that kind of surface is either "flat" or "straight" instead.

Reply:
Let's change it to `Surface.FLAT`